ACPs/204-precompile-secp256r1/README.md

```diff
@@ -0,0 +1,87 @@
+
+This ACP proposes implementing EIP-7212's secp256r1 precompiled contract to unlock significant ecosystem benefits:
+
+### Enterprise & Institutional Adoption
+
+- Reduced onboarding friction: Enterprises can leverage existing biometric authentication infrastructure instead of managing seed phrases or hardware wallets
+- Regulatory compliance: Institutions can utilize their approved device security standards and identity management systems
+- Cost optimization: 100x gas reduction (from 200k-330k to 3,450 gas) makes enterprise-scale applications economically viable
+
+The 100x gas cost reduction makes these use cases economically viable while maintaining the security properties institutions and users expect from their existing devices.
+
+## Specification
+
+This ACP implements [RIP-7212](https://github.com/ethereum/RIPs/blob/master/RIPS/rip-7212.md) for secp256r1 signature verification on Avalanche. The specification follows RIP-7212 exactly, with the precompiled contract deployed at address `0x0200000000000000000000000000000000000006`.
+
+### Core Functionality
+
+- Input: 160 bytes (message hash + signature components r,s + public key coordinates x,y)
+- Output: success: 32 bytes `0x...01`; failure: no data returned
+- Gas Cost: 3,450 gas (based on EIP-7212 benchmarking)
+- Validation: Full compliance with NIST FIPS 186-3 specification
+
+### Activation
+
+This precompile may be activated as part of Avalanche's next network upgrade. Individual Avalanche L1s and subnets could adopt this enhancement independently through their respective client software updates.
+
+For complete technical specifications, validation requirements, and implementation details, refer to [RIP-7212](https://github.com/ethereum/RIPs/blob/master/RIPS/rip-7212.md).
+
+## Backwards Compatibility
+
+This ACP introduces a new precompiled contract and does not modify existing functionality. No backwards compatibility issues are expected since:
+
+1. The precompile uses a previously unused address
+2. No existing opcodes or consensus rules are modified
+3. The change is additive and opt-in for applications
+
+Adoption requires a coordinated network upgrade for the C-Chain. Other EVM L1s can adopt this enhancement independently by upgrading their client software.
+
+## Security Considerations
+
+### Cryptographic Security
+
+- The secp256r1 curve is standardized by NIST and widely vetted
+- Security properties are comparable to secp256k1 (used by ECRECOVER)
+- Implementation follows NIST FIPS 186-3 specification exactly
+
+### Implementation Security
+ 
+- Signature verification (vs public-key recovery) approach maximizes compatibility with existing P-256 ecosystem
+- No malleability check included to match NIST specification, but wrapper libraries should add this
```

Comment:
I feel like documenting an expectation of wrapper libraries to check this is a bit odd.
```suggestion
- No malleability check included to match NIST specification, but wrapper libraries may choose to add this
```
fwiw I don't think there is a problem with allowing malleability here. Just seems like a stronger statement than we should make.